### PR TITLE
Add RKNN object detection docs

### PR DESCRIPTION
**Pull request by mcm001**

### CODE REVIEW

File: source/docs/objectDetection/about-object-detection.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
+About Object Detection
+======================
+
+.. image:: images/notes-ui.png
+
+How does it work?
+^^^^^^^^^^^^^^^^^^^
```
Comment (mdurrani808) [inline source/docs/objectDetection/about-object-detection.rst:7]: underline too long

File: source/docs/objectDetection/about-object-detection.rst [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+PhotonVision supports object detection using neural network accelerator hardware built into Orange Pi 5/5+ coprocessors. The Neural Processing Unit, or NPU, is `used by Photon <https://github.com/PhotonVision/rknn_jni/tree/main>`_ to massively accelerate certain math operations like those needed for running ML-based object detection.
```
Comment (mdurrani808) [inline source/docs/objectDetection/about-object-detection.rst:9]: is used by PhotonVision

File: source/docs/objectDetection/about-object-detection.rst [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+Compared to other pipelines, object detection exposes very few tuning handles. The Confidence slider changes the minimum confidence that the model needs to have in a given detection to consider it valid, as a number between 0 and 1 (with 0 meaning completely uncertain and 1 meaning maximally certain).
+
+.. raw:: html
+
+        <video width="85%" controls>
+            <source src="../../_static/assets/objdetectFiltering.mp4" type="video/mp4">
+            Your browser does not support the video tag.
+        </video>
+
+The same area, aspect ratio, and target orientation/sort parameters from :ref:`reflective pipelines <docs/reflectiveAndShape/contour-filtering:Reflective>` are also exposed in the object detection hard.
```
Comment (mdurrani808) [inline source/docs/objectDetection/about-object-detection.rst:25]: card

File: source/docs/objectDetection/about-object-detection.rst [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+Training Custom Models
+^^^^^^^^^^^^^^^^^^^^^^
+
+Coming soon!
+
+Uploading Custom Models
```
Comment (mdurrani808) [inline source/docs/objectDetection/about-object-detection.rst:32]: big red warning that you cant just slap any model in here and it needs to be yolov5, properly configured, etc. only do this if you know waht youre doing

**Comment by mdurrani808** [comment]
probably just want a little getting started section saying "to get started change the pipeline type to Object Detection" with an admonition about how this won't work on non Opi for parity with the other pipelines

File: source/docs/objectDetection/about-object-detection.rst (hunk 2 above)
Comment (mcm001) [reply source/docs/objectDetection/about-object-detection.rst:9]: unclear how exactly you mean here?

Comment (mdurrani808) [reply source/docs/objectDetection/about-object-detection.rst:9]: replace "is used by Photon" with "is used by PhotonVision"

**Comment by mcm001** [comment]
good call done

**Comment by mdurrani808** [comment]
Still a few more left of Photon over PhotonVision but lgtm.

**Comment by TheTripleV** [comment]
Note that the yolov5 model might have to be trained by the same rknn fork version of yolov5. We trained a yolov5 using ultralytics and the conversion didn't work because of pickling/unpickling issues

**Comment by mcm001** [comment]
Yeah it's apparently quite particular; alex_idk is working on docs

**Pull request merged**